bundles/org.openhab.binding.velbus/src/main/resources/OH-INF/config/8-channel-device-config.xml

```diff
@@ -211,9 +211,71 @@
 			</options>
 			<advanced>true</advanced>
 		</parameter>
-		<parameter name="counter5Unit" type="text">
-			<label>Counter 4 Unit</label>
-			<description>The unit for Counter 4.</description>
```

Comment:
It looks like the `configuration-description` is added in a wrong place and colides with the previous. 
E.g. the counterunit 5-8 is deleted for the above.

Reply:
It was the previous implementation which was wrong. 
I put the 4 new counters for the VMB8IN-20 on the 8channelDeviceWithCounters config-description. (There was a description typo in the copy/past of these 4 "new" counters)
But this would have  impacted the VMB7IN which does not have 8 counters but only 4, and uses this description too.
I've added a new 8channelDeviceWith8Counters config-description for the VMB8IN-20 and removed the 4 ones in the 8channelDeviceWithCounters config-description.
This is why the differences look so strange.